### ISSUE TITLE
Add es6 module build

### ISSUE THREAD

**Issue by anajavi** [mjrussell/redux-auth-wrapper#194]
I made quick&dirty test of building es6 modules of redux-auth-wrapper:
https://github.com/anajavi/redux-auth-wrapper/commit/9297c8cfffec8614f5e58158e3ac8e6bf39b3a46

This can be imported with:
```
import { connectedReduxRedirect } from 'redux-auth-wrapper/es/history3/redirect'

**Comment by anajavi** [comment]
That problem would need lots of testing and head scratching. It might be that tree shaking in webpack takes care of that, but there might be cases where that does not work.

Maybe I just submit this as a PR?

**Comment by mjrussell** [comment]
Yeah agree, would be tricky. Submitting the PR sounds great, I'll try to
test it out soon.

**Comment by anajavi** [comment]
Hello again,

I'm doing spring cleanup of my github repositories:) Do you think this PR could be merged or should we close it?
